server/handlers/database_handlers.go

```diff
@@ -100,3 +105,76 @@ func (h *Handler) GetSystemDatabase(w http.ResponseWriter, r *http.Request, _ *m
 	}
 	fmt.Fprint(w, string(val))
 }
+
+func (h *Handler) ResetSystemDatabase(w http.ResponseWriter, r *http.Request, _ *models.Preference, _ *models.User, provider models.Provider) {
+
+	mesherydbPath := path.Join(utils.GetHome(), ".meshery/config")
+	err := os.Mkdir(path.Join(mesherydbPath, ".archive"), os.ModePerm)
+	if err != nil && os.IsNotExist(err) {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
```

Comment:
wrap this error in something user friendly for it will be displayed on UI. Also return after this and return everywhere below this wherever you send the HTTP response.

Reply:
Done, thanks!